radioapp/zeroconf.py

```diff
@@ -0,0 +1,20 @@
+from zeroconf import ServiceBrowser, Zeroconf
+import threading
+
+
+class LmrListener(object): #declare this before calling PairingScan
+
+    def __init__(self):
+        self.stop_searching = threading.Event()
+
+    def add_service(self, zeroconf, type, name):
+        info = zeroconf.get_service_info(type, name)
+        print("Service %s added, service info: %s" % (name, info))
+
+def PairingScan(listener, wait_time=2):
```

Comment:
There's no way to actually use the scan information... There should be a way to register a callback or something along that line.

Reply:
What kind of information are you wanting?

Reply:
Currently when start, stop or PairingScan are called, nothing happens when they are finished. We could use a callback to accomplish this. For Example: 

def Start(callback) : 
    the code that start accomplishes
    callback()

In some other function, 

//I want to call start then do this function

def whatIWantToDoAfterStart():
     start is done ill do this

Start(whatIWantToDoAfterStart)